components/modal/Modal.css

```diff
@@ -0,0 +1,79 @@
+@import url("/components/button/Button.css");
+
+.modal-container {
+  position: fixed;
+  display: none;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  left: 0;
+  top: 0;
+  z-index: 99;
+  background-color: rgba(0, 0, 0, 0.2);
+}
+
+.modal {
+  background-color: white;
+  box-shadow: 0 10px 20px 1px rgba(6, 10, 36, 0.1),
+    0 24px 36px 1px rgba(6, 10, 36, 0.18);
+  width: 30rem;
+  height: max-content;
+  border-radius: 1.4rem;
+}
+
+.modal-header {
+  font-size: 1.8rem;
+  background-color: var(--primary-color);
+  color: white;
+  padding: 1rem;
+  border-radius: 1.4rem 1.4rem 0 0;
+  font-weight: bold;
+  display: flex;
+  justify-content: space-between;
+}
+
+.modal-header-icon {
+  cursor: pointer;
+}
+
+.modal-body {
+  color: var(--dark-gray);
+  padding: 1rem;
+}
+
+.modal-footer {
+  display: flex;
+  justify-content: flex-end;
+  padding: 1rem;
+}
+
+.modal-footer-btn {
+  cursor: pointer;
+  margin: 0 5px;
+  padding: 0.5rem;
+  border-radius: 0.4rem;
+  border: none;
+  background-color: var(--primary-color);
+  color: white;
+  font-weight: bold;
+}
+
+.modal-footer-btn:hover {
+  background-color: var(--secondary-color);
+}
+
+.modal-footer-btn-outline {
```

Comment:
suggestion:  you can use button from button components , also if possible

Reply:
Yeah, reusability. Sure, thanks.